spec/models/user_spec.rb

```diff
@@ -40,94 +44,182 @@
   end
 
   describe 'is_admin' do
+    it 'is aliased to is_admin?' do
+      allow(subject).to receive(:groups).and_return(User::ADMIN_GROUPS)
+      expect(subject).to respond_to('is_admin?')
+      expect(subject.is_admin).to be true
+      expect(subject.is_admin?).to be true
+      expect(subject.is_admin).to be subject.is_admin?
+    end
```

Comment:
This spec addresses concerns of @jmartin-sul and the same pattern is applied for the other aliases.